### ISSUE TITLE
CI - Missing Lint Artifacts (And CheckStyle)

### ISSUE THREAD

**Issue by ParaskP7** [wordpress-mobile/AztecEditor-Android#964]
### TL;DR

As part of [Gradle 7.4 & AGP to 7.1.1](https://github.com/wordpress-mobile/AztecEditor-Android/pull/963) upgrade, I noticed that only the `aztec` module related Lint set of artifacts are being uploaded during a CI build (see `~/lint/lint-results-release.html/txt/xml`). However, there are five more Lint related artifacts that need to be uploaded as well, for the `wordpress-shortcodes`, `wordpress-comments`, `picasso-loader`, `glide-loader` and `app` modules.

PS: The same actually applies to CheckStyle related artifacts.

### Details

This is most probably due to the way the `Save lint results` and `Uploading artifacts` works, which copies any Lint related artifacts that match the `.*/build/reports/.*` regex, thus overriding previous set of artifacts. This script probably expects that a repo would only contain one set of Lint related artifacts. But, for this repo we are actually interested in four sets of Lint related artifacts, for the below modules:

- `app` (App)
- `aztec` (Library)
- `glide-loader` (Library)
- `picasso-loader` (Library)
- `wordpress-comments` (Library)
- `wordpress-shortcodes` (Library)

Thus, and since the `aztec` probably runs last, this set of Lint artifacts overrides any preceding sets of Lint artifacts, leaving only those to be then uploaded.

PS: The exact same applies to the CheckStyle artifacts.
